services/kafka/service.go

```diff
@@ -318,21 +315,23 @@ func (s *Service) Test(options interface{}) error {
 	if !ok {
 		return fmt.Errorf("unknown cluster %q", o.Cluster)
 	}
-	return c.WriteMessage(s.diag, o.Topic, []byte(o.Key), []byte(o.Message))
+	return c.WriteMessage(s.diag, WriteTarget{Topic: o.Topic}, []byte(o.Key), []byte(o.Message))
```

Comment:
I figure people don't really need to specify the partitioning details when they're just testing the integration